docs/Makefile

```
@@ -115,6 +115,18 @@ test: doctest
 deps:
 	pip3 install -r requirements-docs.txt
 
+deps-test:
+	pip3 install -r requirements-docs-test.txt
+
+# Check that all the dependencies have been installed
+# in release mode, this means they should be installed
+# from the wheels
+check-deps:
+	pip3 show oso
+	pip3 show django-oso
+	pip3 show flask-oso
+	pip3 show sqlalchemy-oso
```

Adapted from https://stackoverflow.com/a/65606063
```suggestion
	python3 -c "import pkg_resources; pkg_resources.require(open('requirements-docs-test.txt'))"
```
This way we only have to update deps in one place.

---

Ah no this doesn't work, because they are all `-e .../` links

---

This will all be gone soon, so its fine